getting-started/setup-multi-node-basic.md

```diff
@@ -0,0 +1,110 @@
+the architecture can be found [here][architecture]. The steps for 
+creating a basic multi-node setup are as follows:
+
+## Basic setup
+1. Configure data nodes for communication with the access node
+1. Initialize data nodes from the access node 
+
+### Prerequisites
+- One PostgreSQL instance to act as an access node
+- One or more PostgreSQL instances to act as data nodes
+- TimescaleDB [installed][install] and [set up][setup] on all instances
+- Access to a superuser role (e.g. `postgres`) on all instances
+
+### 1. Configure data nodes for node-to-node communication
+To enable communication between the access node and the data node(s), the 
+underlying PostgreSQL instances for each data node must be configured to 
+authorize external communication.  The simplest way is to use "trust" 
+authentication which provides unencumbered client access to the instances.
+This method must be applied to each PostgreSQL instance intended to host 
+a data node.
+
+>:WARNING: This authentication method allows insecure access to all 
+instances.  For production implementations, please use more secure 
+methods of authentication (see [advanced setup][] for examples).
+
+#### Edit authentication configuration file
+Client authentication for PostgreSQL instances is usually configured in 
+the `pg_hba.conf`([reference doc][postgresql-hba]) file located in the 
+data directory.  If the file is not located there, access the instance
+(psql) and execute the command:
+
+```sql
+SHOW hba_file;
+``` 
+
+To enable "trust" authentication, add a line to `pg_hba.conf` to allow the
+`postgres` superuser to access the instance:
+
+```
+# TYPE  DATABASE  USER      ADDRESS       METHOD
+host    all       postgres  {instance_ip}  trust
+```
+
+where `instance_ip` is the ip address for the respective instance.
+
+#### Edit main configuration
+It is necessary to change the parameter `max_prepared_transactions` to a 
+non-zero value if it hasn't been changed already.  The recommended value 
+is 150.The parameter is located in `postgresql.conf`, typically in the 
+data directory.  If it isn't there, access the instance(psql) and get the 
+path with:
+```
+SHOW config_file;
+```
+
+##### Reload server configuration
+
+To reload the server configuration, you can use the following command
+on the data node:
+
+```
+pg_ctl reload
+```
+
+### 2. Initialize data nodes from the access node
+From within the access node instance(psql), use the command:
+```sql
+SELECT add_data_node('{node_name}', host => '{node_ip_address}');
+```
+`{node_name}` should be a unique name for the node.  `{node_ip_address}` 
+is the ip address for the node (host and port, if necessary).
+
+The system should be up and running at this point, ready to be used.
+
+---
+## Next steps
+To start working with the system, you can look at documentation for [distributed hypertables][].
+
+To further configure the system (set up secure node-to-node communication, add 
+additional users/roles) see [advanced setup][].
```

Comment:
Maybe I missed it, but I didn't see anything about setting up additional users under "Advanced setup". I would expect to see something about using `distributed_exec` to create roles, etc.